modules/drivers/snowflake/src/metabase/driver/snowflake.clj

```diff
@@ -283,12 +284,12 @@
                                               (qp.store/fetch-and-store-database! (u/the-id database))
                                               (sql.qp/->honeysql driver table))]}))
 
-(defmethod driver/describe-database :snowflake
-  [driver database]
+(defn describe-database* [database]
```

Comment:
Might make sense to keep the `driver` parameter, so that drivers inheriting from snowflake can profit from it.